stdlib/public/Concurrency/Task+immediate.swift.gyb

```diff
@@ -19,14 +19,15 @@ import Swift
 %   'THROWING',
 %   'NON_THROWING',
 % ]
-% OPERATION_PARAM = '@_inheritActorContext @_implicitSelfCapture _ operation: __owned @Sendable @escaping () async throws -> Success'
 % for METHOD_VARIANT in METHOD_VARIANTS:
 
 % IS_THROWING = METHOD_VARIANT == 'THROWING'
 % if IS_THROWING:
 %   FAILURE_TYPE = 'Error'
+%   THROWS = 'throws '
```

Comment:
is the space after the keyword doing something meaningful?

Reply:
Kinda, the way we insert it into the template so it doesn't touch something else. But it doesn't matter tbh, we could just give extra space in the template. No one will be looking at the produced source